### PR TITLE
[DOCFIX] Add documentation for remote logging

### PR DESCRIPTION
**Pull request by riversand963**
@aaudiber @gpang 
Could you take a look? Thanks!

### CODE REVIEW

File: docs/en/Remote-Logging.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+Suppose the hostname of the log server is AlluxioLogServer, and the port is 45010.
+Refer to [Running Alluxio on a cluster](Running-Alluxio-on-a-Cluster.html) for instructions in deploying
+Alluxio in a cluster.
+
+By default, remote logging is not enabled. To enable Alluxio remote logging, you can either set a
+few environment variables or modify the JVM properties directly.
+
+This section describes these two different approaches step by step. There is no requirement on where
+the log server must run, as long as the other Alluxio servers have access to it. In our example, we
+run the log server on the same machine as the primary master. If you want to run the log server on
```
Comment (aaudiber) [inline docs/en/Remote-Logging.md]: there isn't a designated primary master; the master which is primary could change at any time. We can only say "on the same machine as a master"

File: docs/en/Remote-Logging.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+{:toc}
+
+## Overview
+Alluxio supports sending logs to a remote log server over the network. This feature can be useful
+to system administrators who have to perform the task of log collection. With remote logging, the
+log files, e.g. master.log, worker.log, etc. on all Alluxio servers will be readily available on
+a designated and configurable directory on the log server.
+
+## Configure Alluxio
+Suppose the hostname of the log server is AlluxioLogServer, and the port is 45010.
```
Comment (aaudiber) [inline docs/en/Remote-Logging.md]: Define the hostname and log server port closer to where they are used, or let the context explain what they mean

File: docs/en/Remote-Logging.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+Refer to [Running Alluxio on a cluster](Running-Alluxio-on-a-Cluster.html) for instructions in deploying
+Alluxio in a cluster.
+
+By default, remote logging is not enabled. To enable Alluxio remote logging, you can either set a
+few environment variables or modify the JVM properties directly.
+
+This section describes these two different approaches step by step. There is no requirement on where
+the log server must run, as long as the other Alluxio servers have access to it. In our example, we
+run the log server on the same machine as the primary master. If you want to run the log server on
+another machine, you need to copy the Alluxio directory to the desired machine (with `rsync`
```
Comment (aaudiber) [inline docs/en/Remote-Logging.md]: I don't think it's necessary to specify that Alluxio needs to be installed before you can run its services - anyone reading this guide should know that already.

File: docs/en/Remote-Logging.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+### Enable Remote Logging with Environment Variables
+In ./conf/alluxio-env.sh, add the following lines:
+
+```bash
+ALLUXIO_LOGSERVER_HOSTNAME=AlluxioLogServer
+ALLUXIO_LOGSERVER_PORT=45010
+ALLUXIO_LOGSERVER_LOGS_DIR=/tmp/alluxio_remote_logs
+```
+
+### Enable Remote Logging with JVM Properties
```
Comment (aaudiber) [inline docs/en/Remote-Logging.md]: When should users use this approach instead of the Environment Variables approach?

Comment (riversand963) [reply docs/en/Remote-Logging.md]: This approach is less intuitive than the environment variable approach, but I think some advanced users might want to know this possibility.

File: docs/en/Remote-Logging.md (hunk 3 above)
Comment (riversand963) [reply docs/en/Remote-Logging.md]: Do we need to emphasize that the alluxio directory on all the machines should have the same `conf` and `libexec` directory? I think sometimes, users may forget syncing the folders across the machines.

File: docs/en/Remote-Logging.md (hunk 1 above)
Comment (riversand963) [reply docs/en/Remote-Logging.md]: Done.

File: docs/en/Remote-Logging.md (hunk 2 above)
Comment (riversand963) [reply docs/en/Remote-Logging.md]: Done.

File: docs/en/Remote-Logging.md (hunk 3 above)
Comment (aaudiber) [reply docs/en/Remote-Logging.md]: I thought only the log server port needs to match. As long as we don't break log server backwards compatibility between versions, I could imagine users upgrading their masters and workers to a new version without upgrading their log server.

File: docs/en/Remote-Logging.md (hunk 4 above)
Comment (aaudiber) [reply docs/en/Remote-Logging.md]: If we give the user two ways to do something, the first thing they want to know is which one is right for them. We need to make it clear in the doc when to use which approach

Comment (riversand963) [reply docs/en/Remote-Logging.md]: Done.

File: docs/en/Remote-Logging.md (hunk 3 above)
Comment (riversand963) [reply docs/en/Remote-Logging.md]: Yeah, we support that.

**Pull request merged**